lib/PerfMeter.js

```diff
@@ -139,7 +139,7 @@ PerfMeter.prototype = {
 
   displayItem(tabId, item) {
     if (this._active) {
-      console.info(`${tabId} ${item.start} ${item.tag}${item.data ? ":" + item.data : ""} ${item.delta || ""}`);
+      console.info(`${tabId} ${item.start} ${item.tag}${item.data ? ":" + item.data : ""} ${item.delta || ""}`); // eslint-disable-line no-console
```

Comment:
Your suggestion and what's implemented are not 1-1, though I like the brevity of what you suggested.

In the case `item.data` is falsey, the code would print: `[TAB_ID] [ms_elapsed] [some-tag]`
Otherwise, the code will print `[TAB_ID] [ms_elapsed] [some-tag]:[event-data]`

Note the `:` between the event tag and the data.